Description: allow min() and max() in expresion, by Luca Marturana
Origin: upstream, http://lybniz2.svn.sourceforge.net/viewvc/lybniz2/trunk/lybniz.py?view=patch&r1=104&r2=103&pathrev=104

--- a/lybniz.py
+++ b/lybniz.py
@@ -81,6 +81,8 @@
 
 #add any needed builtins back in.
 safe_dict['abs'] = abs
+safe_dict['min'] = min
+safe_dict['max'] = max
 
 def marks(min_val,max_val,minor=1):
 	"yield positions of scale marks between min and max. For making minor marks, set minor to the number of minors you want between majors"
